app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -358,19 +360,65 @@ class TopicFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicFragment_withStoryId_clickOnPracticeTab_configChange_showsSameTabAndItsContent() {
```

Comment:
why we are saying `withStoryId`? I think we can skip this.

Reply:
we are using `withStoryId` because in all other test cases we are not passing `storyid` as an argument and without passing `storyId` we can't go to lessons tab.

Reply:
I think keeping `withStoryId` is correct here because based on that flow of the tabs is decided.